Assets/_Game/Scripts/Events/EventManager.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EventManager
+{
+    public delegate void D_Void();
+    public delegate void D_Int(int value);
+    public delegate void D_GameObjectInt(GameObject value, int value2);
+    public delegate void D_ItemWithInventoryWithInt(InventoryObject inventory, ItemObject item, int amount);
+    public delegate void D_ItemWithInventory(InventoryObject inventory, ItemObject item);
+    public delegate void D_Inventory(InventoryObject inventory);
+    public delegate void D_ItemWithGameObject(ItemObject item, GameObject obj);

```

Comment:
Suggestion: Probably should remove delegates you aren't actively using

Reply:
yep need to clean that up